classes/local/object_manipulator/candidates/deleter_candidates.php

```diff
@@ -34,17 +34,15 @@ class deleter_candidates extends manipulator_candidates_base {
      * @return string
      */
     public function get_candidates_sql() {
-        return 'SELECT MAX(f.id),
-                       f.contenthash,
-                       MAX(f.filesize) AS filesize
-                  FROM {files} f
-                  JOIN {tool_objectfs_objects} o ON f.contenthash = o.contenthash
-                 WHERE o.timeduplicated <= :consistancythreshold
-                   AND o.location = :location
-                   AND f.filesize > :sizethreshold
-              GROUP BY f.contenthash,
-                       f.filesize,
-                       o.location';
+        return 'SELECT contenthash,
```

Comment:
The first column must be guaranteed to be unique but we still have a GROUP BY here, can we eliminate the GROUP BY completely now tha we don't have the join?

I think this applies to all the the sql in the same way